DQM/MuonMonitor/BuildFile.xml

```diff
@@ -0,0 +1,21 @@
+<use   name="Geometry/CSCGeometry"/>
```

Comment:
@calderona are all these dependencies really needed? Where are digis rechits and geometry used for instance?

Reply:
Hi @fabiocos 
sorry, I thought I have answered this question already. I modified the BuildFile: https://github.com/cms-sw/cmssw/blob/3c1ab03ea63d4662d5488e7ac2d9d8573931c2e3/DQM/MuonMonitor/plugins/BuildFile.xml
It is true still there are some files for digi, so I will remove them.